src/main/scala/io/snyk/plugin/client/ApiClient.scala

```diff
@@ -72,41 +79,33 @@ private final class StandardApiClient(tryConfig: => Try[SnykConfig]) extends Api
     }
   }
 
-  private def runRaw(treeRoot: SnykMavenArtifact): Try[String] = tryConfig flatMap { config =>
-    val jsonReq = stringifyWithoutNulls(SnykClientSerialisation.encodeRoot(treeRoot, config.org))
+  private def runRaw(project: Project, treeRoot: SnykMavenArtifact): Try[String] = tryConfig flatMap { config =>
+    log.debug("ApiClient: run Snyk CLI")
 
-    log.debug("ApiClient: Built JSON Request")
-    log.debug(jsonReq)
+    val projectPath = project.getBasePath
 
-    val apiEndpoint = config.endpointOrDefault
-    val apiToken = config.api
```

Comment:
we don't use the token anymore, so a user has to run `snyk auth` first, correct?

Reply:
Good point Pavel. Thanks!